src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -117,7 +120,7 @@ public boolean hasTarget() {
      *         are degrees).
      */
     public double getVisionYaw() {
-        return shooterVisionSubsystem.getYaw();
+        return yawPass.calculate(shooterVisionSubsystem.getYaw()*0.5);
```

Comment:
Why is this scaled by 1/2? (And why scale here and not in ShooterVisionSubsystem?)

Reply:
Was done to slow down turret before realizing the image was flipped, undoing